tensorflow/.buildspec_templates/std_buildspec_template.yml

```diff
@@ -0,0 +1,94 @@
+  account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
is there any specific reason to make these templates files as hidden?

Reply:
It's intended to reduce clutter.